test-app/src/widgets/Draggable.js

```diff
@@ -92,12 +101,6 @@ export default class Draggable extends createjs.Container {
   onDragStart(evt) {
     evt.stopPropagation();
     evt.preventDefault();
-    // if draggable is picked when its already in drop zone
```

Comment:
onDragStart is only fired when using a mouse, so I pulled this into removeFromPrevTarget to use in drop and revert, which are common to mouse and KN controls